src/main/webapp/WEB-INF/jsp/scoreboard.jsp

```diff
@@ -228,6 +228,7 @@
                             <div class="panel-body game-clock">
                                 <div class="digits-alt">
                                     <span id="gameClockMins">${score.gameClock.mins}</span>:<span id="gameClockSecs">${score.gameClock.secs}</span>
+                                    <!--<span id="gameClockMins">05</span>:<span id="gameClockSecs">35</span>-->
```

Comment:
Nah, that's just left over. I'll get rid of that for the next commit.  This was just while i was playing with CSS so that i don't have to go to scorrer page on every refresh